app/views/administrate/application/edit.html.erb

```diff
@@ -15,13 +15,13 @@ It displays a header, and renders the `_form` partial to do the heavy lifting.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Form
 %>
 
-<% content_for(:title) { "Edit #{page.page_title}" } %>
+<% content_for(:title) { "#{t("administrate.actions.edit")} #{page.page_title}" } %>
```

Comment:
Agreed, something like `t("administrate.actions.edit_resource", resource: page.page_title)` would be good.